fastlane/lanes/release.rb

```diff
@@ -56,12 +54,13 @@
       release_notes_file_path: changelog_path
     )
 
-    generate_strings_file_for_glotpress
-
     UI.important('Pushing changes to remote, configuring the release on GitHub, and triggering the beta build...')
     UI.user_error!("Terminating as requested. Don't forget to run the remainder of this automation manually.") unless skip_confirm || UI.confirm('Do you want to continue?')
 
-    push_to_git_remote(tags: false)
+    push_to_git_remote(
+      tags: false,
+      set_upstream: is_ci == false # only set upstream when running locally, useless in transient CI builds
```

Comment:
For my learning, why do we only want to set the upstream when running locally?

Reply:
Otherwise, we'll find ourselves on a `release/x.y` branch that does not point to `origin` and we won't be able to push pending configuring the upstream ourselves. It's just a convenience.